src/services/companyService.js

```diff
@@ -23,7 +23,7 @@ export default class CompanyService {
       });
       return result;
     } catch (err) {
-      return new Error('failed to create company');
+      throw new Error('failed to create company');
```

Comment:
I feel this should be
```suggestion
      throw new ApiError(400 || 500, 'failed to create company');
```
Since we have a custom ApiError for the App, we should use it to throw errors

Reply:
Thanks, @Segun-Ogundipe. I would update the PR as advised.